pubobot/bot.py

```diff
@@ -143,7 +143,7 @@ def __contains__(self, player):
 
 
 class Match:
-    def __init__(self, pickup, players):
+    def __init__(self, pickup, players: List[discord.Member]):
```

Comment:
I noticed that Python, or at least this code, is very averse to typing. I added types for player lists in a few places to help my own sanity

Reply:
Typing was likely introduced before this code was written.